ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -219,6 +221,20 @@ parseCommand
    : PARSE (source_field = expression) (pattern = stringLiteral)
    ;
 
+rexCommand
```

Comment:
add new keyword to keywordsCanBeId list

Reply:
It looks like EXTRACT is already properly included in the keywordsCanBeId lists in all the relevant PPL parser grammar files. 

`/ppl/src/main/antlr/OpenSearchPPLParser.g4)`: `EXTRACT` is included at line 1236: https://github.com/opensearch-project/sql/blob/994bdd5588ab5fa5b2bfd897ace786fc0c046d14/ppl/src/main/antlr/OpenSearchPPLParser.g4#L1236